src/components/logicComponents/Carousel.js

```diff
@@ -3,22 +3,34 @@ import './styles.scss';
 import { Link } from 'react-router-dom';
 import { useHorizontalScroll } from 'hooks/useSideScroll';
 import LOADER from 'imgs/loader.gif';
+import { BsFillBookmarkFill } from 'react-icons/bs';
 
 export function Carousel({ movies }) {
   const scrollRef = useHorizontalScroll();
   return (
     <div ref={scrollRef} className="Carousel-container">
-      {movies && movies.length !== 0 ? (
-        movies.map(movie => (
-          <Link to={`/movie/${movie.id}`}>
-            <Movie
-              key={movie.id}
-              img={movie.backdrop_path}
-              title={movie.original_title}
-              date={movie.release_date}
-            />
-          </Link>
-        ))
+      {movies !== undefined && movies.length !== 0 ? (
```

Comment:
```suggestion
      {!movies.length ? (
```

https://www.freecodecamp.org/news/check-if-javascript-array-is-empty-or-not-with-length/#:~:text=To%20check%20if%20an%20array,it%20is%20empty%20or%20not.

Reply:
same here too